locales/de_DE.json

```diff
@@ -1,37 +1,70 @@
 {
   "sandbox-warning": "This is a sandbox environment for testing API integration. Never send real funds here!",
+  "Sandbox-Warnung": "Das ist eine Sandbox-Umgebung zum testen der API-Integration. Senden Sie niemals echtes Geld (Kryptowährung) hierher",  
   "buy": "buy",
+  "Kaufen": "Kaufen",
   "sell": "sell",
+  "Verkaufen": "Verkaufen",
   "continue": "Continue",
+  "Fortfahren: "Fortfahren",
   "ok": "OK",
+  "Ok": "OK",
   "back": "Back",
+"Zurück": "Zurück",
```

Comment:
This is not how it works. The left side is the key and needs to stay in English. The right side, after the semicolon, needs to be the translated word.

So: it should be:
`"back": "Zurück",`

Reply:
Oh thanks i will change it :+1: